src/Summoner/Tui/Kit.hs

```diff
@@ -142,21 +150,23 @@ summonKitToSettings sk = Settings
     , settingsLicenseName    = sk ^. project . license
     , settingsLicenseText    = ""
     , settingsGitHub         = isGitHub
+    , settingsPrivate        = isGitHub && sk ^. gitHub . private
     , settingsTravis         = isGitHub && sk ^. gitHub . travis
     , settingsAppVeyor       = isGitHub && sk ^. gitHub . appVeyor
     , settingsIsLib          = sk ^. projectMeta . lib
     , settingsIsExe          = sk ^. projectMeta . exe
     , settingsTest           = sk ^. projectMeta . test
     , settingsBench          = sk ^. projectMeta . bench
-    , settingsTestedVersions = sk ^. projectMeta . ghcs
+    , settingsTestedVersions = sortNub $ defaultGHC : (sk ^. projectMeta . ghcs)
     , settingsBaseType       = baseT
     , settingsPrelude        = cP
-    , settingsExtensions     = []
-    , settingsWarnings       = []
+    , settingsExtensions     = sk ^. extensions
+    , settingsWarnings       = sk ^. warnings
     , settingsCabal          = sk ^. cabal
     , settingsStack          = sk ^. stack
-    , settingsStylish        = Nothing
-    , settingsContributing   = Nothing
+    , settingsStylish        = const "" <$> (sk ^. stylish)
+    , settingsContributing   = const "" <$> (sk ^. contributing)
```

Comment:
Maybe the following will work:

```haskell
"" <$ sk  ^. contributing
```

Though, unprepared code readers might not be happy about such code...